### PR TITLE
Intake

### LINKED ISSUE #6: Make Harvester
Make a Harvester class, complete with: 
- [x] a collector (VictorSP) and
- [x] a deployer (Solenoid)

### PR DESCRIPTION
**Pull request by cadenmr**
Merge Xander's intake code. Closes #6.

### CODE REVIEW

File: src/main/java/frc/robot/Intake.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    //ESC declarations
+    private final VictorSP intakeMotor; 
+
+    //Piston declarations
+    private final Solenoid expanderPiston;
+
+
+    //class initializer
+    public Intake(int intakeMotorPort, int expanderPistonPort) {
+
```
Comment (TaeSeanDo) [inline src/main/java/frc/robot/Intake.java]: Unnecessary whitespace on lines 17, 20, and 33.

File: src/main/java/frc/robot/Intake.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    //Setters
+
+    //ESC
+    public void intakeMotorControl(double power) {
+        intakeMotor.set(power);
+    }
+
+    //Piston
+    public void expanderPistonControl(boolean state) {
```
Comment (TaeSeanDo) [inline src/main/java/frc/robot/Intake.java]: Rename state to out for clarity of the true/false convention.

File: src/main/java/frc/robot/Intake.java [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    public Intake(int intakeMotorPort, int expanderPistonPort) {
+        
+        intakeMotor = new VictorSP(intakeMotorPort);
+        expanderPiston = new Solenoid(expanderPistonPort);
+
+    }
+
+    //Setters
+
+    //ESC
```
Comment (TaeSeanDo) [inline src/main/java/frc/robot/Intake.java]: I would like a clarifying comment that positive power spins inward.

File: src/main/java/frc/robot/Intake.java [hunk 4]
```diff
@@ -0,0 +1,35 @@
+package frc.robot;
+
+import edu.wpi.first.wpilibj.VictorSP;
+import edu.wpi.first.wpilibj.Solenoid;
+
+public class Intake {
```
Comment (TaeSeanDo) [inline src/main/java/frc/robot/Intake.java]: Please rename class to Harvester. I just like that name better.

File: src/main/java/frc/robot/Intake.java [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+    //ESC declarations
+    private final VictorSP intakeMotor;
```
Comment (docprofsky) [inline src/main/java/frc/robot/Intake.java]: While we are correcting whitespace, let us also ensure there is a space between the `//` and the body of the comment.
Example: `// ESC declarations`
